tests/testthat/fixtures/vb/make_vb.R

```diff
@@ -1,24 +1,24 @@
 #Define specific functions for chosen DE, distribution of parameters, error process
 #Differential equation governing dynamics.
 DE <- function(y, pars){ #Canham function
-  return(pars[1] * (pars[2] - y))
+  return(pars[2] * (pars[1] - y))
```

Comment:
Just checking: You sure about swapping the index here?

Reply:
Yes, max size is the first parameter and the growth rate is the second.